common-information/common-localization.md

```diff
@@ -318,17 +322,59 @@ Of course, if you don't want to hard-code your translation inside your source co
 
 All that's left to do is to set our CustomLocalizationManager to the static Manager property of the LocalizationManager. Please note that you need to do this assingnment prior to invoking the InitializeComponent method of the affected controls.
 
-#### __[C#] {% if site.site_name == 'WPF' %}Example 5{% endif %}{% if site.site_name == 'Silverlight' %}Example 7{% endif %}: Applying the custom LocalizationManager__
+#### __[C#] {% if site.site_name == 'WPF' %}Example 5{% else %}Example 7{% endif %}: Applying the custom LocalizationManager__
 {{region common-localization_0}}
 	LocalizationManager.Manager = new CustomLocalizationManager();
-        InitializeComponent();
+    InitializeComponent();
 {{endregion}}
 
-#### __[VB.NET] {% if site.site_name == 'WPF' %}Example 5{% endif %}{% if site.site_name == 'Silverlight' %}Example 7{% endif %}: Applying the custom LocalizationManager__
+#### __[VB.NET] {% if site.site_name == 'WPF' %}Example 5{% else %}Example 7{% endif %}: Applying the custom LocalizationManager__
 {{region common-localization_1}}
 	LocalizationManager.Manager = New CustomLocalizationManager()
 	InitializeComponent()
 {{endregion}}
 
+# LocalizableResourceExtension
+
+The LocalizableResourceExtension is a handy markup extension that returns a localization string based on a given key.
+
+#### __[XAML] {% if site.site_name == 'WPF' %}Example 6{% else %}Example 8{% endif %}: Using the LocalizableResourceExtension__
+{{region common-localization_12}}
+	<telerik:RadButton Content="{telerik:LocalizableResource Key=GridViewClearFilter}" Click="GridView_ClearFilter" />
+{{endregion}}
+
+In the example above, it uses the GridViewClearFilter key provided by the default ResourceManager of the LocalizationManager, however, you can easily use your own keys by specifying a [custom ResourceManager](#localization-using-resourcemanager).
+
+#### __[XAML] {% if site.site_name == 'WPF' %}Example 7{% else %}Example 9{% endif %}: Get custom localization string__
+{{region common-localization_13}}
+	<telerik:Label Content="{telerik:LocalizableResource Key=EmployeeString}" />
+{{endregion}}
+
+## Dynamic Localization
```

Comment:
This is not supported by all controls. Please add a note somewhere that mention this. Also, can add a list with controls that support this.